_layouts/event.html

```diff
@@ -10,27 +10,25 @@
   <header class="post-header">
     <h1 class="post-title p-name" itemprop="name headline">{{ page.title | escape }}</h1>
     <p class="post-meta">
-      <time class="dt-published" datetime="{{ page.date | date_to_xmlschema }}" itemprop="datePublished">
-        {%- assign date_format = site.minima.date_format | default: "%b %-d, %Y" -%}
-        {{ page.date | date: date_format }}
-      </time>•
-      {%- for information in layout.dial_in_information -%}
-        <span itemprop="author" itemscope itemtype="http://schema.org/Person"><span class="p-author h-card" itemprop="name"> {{ information }} •</span></span>
-      {%- endfor -%}
-        <span itemprop="author" itemscope itemtype="http://schema.org/Person"><span class="p-author h-card" itemprop="name"> <a href="{{ layout.dial_in_link }}">Dial In</a></span></span>
+        <span itemprop="author" itemscope itemtype="http://schema.org/Person"><span class="p-author h-card" itemprop="name"> Zoom Dial In Link: <a href="{{ layout.dial_in_link }}">{{ layout.dial_in_link }}</a></span></span>
       </p>
   </header>
 
   <div class="post-content e-content" itemprop="articleBody">
     {{ content }}
-  </div>
-  <h2>Recording</h2>
-    {{ page.recording }}
-    <h2>Presentation</h2>
-    {{ page.presentation }}
+    {%- if page.file_list.size > 0 -%}
     <h2>Files</h2>
-    {%- for page_file in page.file_list -%}
-      •<a href="{{ page_file }}">{{ page_file }}</a> <br>
-    {%- endfor -%}
+    <ul>
+      {%- for page_file in page.file_list -%}
+        {%- if page_file[0] contains '.com' or page_file[0] contains '.edu' or page_file[0] contains '.org' -%}
+          <li><a href="{{ page_file[0] }}">{{ page_file[0] }}</a>: {{page_file[1]}} </li>
+        {% else -%}
+        {% assign file_store = "/file_store/" %}
+          <li><a href="{{  file_store | append:  page_file[0] | append: site.baseurl }}">{{ page_file[0] }}</a>: {{page_file[1]}} </li>
```

Comment:
Does this url construction actually work? It's a bit confusing.

Reply:
as far as i can tell, it does

Reply:
I guess I'm not convinced the append makes things clearer... I might be missing something?
```
{{  file_store }}{{ page_file[0] }}{{  site.baseurl }}
```